docs/data/indexing/data-indexers/README.mdx

```diff
@@ -1,9 +1,9 @@
 ---
 sidebar_position: 10
-title: Data Indexer
+title: Ecosystem Data Indexers
 ---
 
-## Data Indexers
+## Ecosystem Data Indexers
```

Comment:
I think we should kill this page and just have the sidebar link to the "providers" page. See like how it is done in the "Hubble" and "Oracles" views.